frontend/src/bundles/common/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate } from '~/bundles/common/components/components.js';
+import { AppRoute } from '~/bundles/common/enums/enums.js';
+
+interface Properties {
+    children: React.ReactNode;
+}
+
+const ProtectedRoute: React.FC<Properties> = ({ children }) => {
```

Comment:
we already have JWT

Reply:
It might be necessary to create a ticket to update that so we don't do it here in the Not Found Page task.